lib/config/sass-lint.yml

```diff
@@ -25,6 +25,7 @@ rules:
   no-css-comments: 1
   no-debug: 1
   no-disallowed-properties: 0
+  no-domains: 0
```

Comment:
Sounds good to me. I didn't know whether or not we wanted to suppress this rule until by default until 2.0.  Added as a warning by default.